command/sub8_mission_control/action/current_goal.action

```diff
@@ -0,0 +1,8 @@
+#goal definition
+nav_msgs/Odometry order
+---
+#result definition
+bool flag
+---
+#feedback
+int32[] sequence
```

Comment:
At the moment this is just leftover from the default action file template. Right now feedback is not implemented, only the goal and flag are. That will change when I add the server.